Source/WebInspectorUI/UserInterface/Views/ScopeBar.css

```diff
@@ -40,7 +40,7 @@
     display: inline-block;
     margin: 0 var(--scope-bar-margin);
     padding: 2px var(--scope-bar-padding) 4px;
-    font-size: 11px;
+    font-size: 0.7rem;
```

Comment:
this will likely have much more far reaching consequences than just in the Audit Tab since `WI.ScopeBar` is used all over the place

also, what is the benefit of switching from `px` to `rem`?

Reply:
I am going to draft this until command +/- is implemented in Web Inspector.

Reply:
It will not make too much of a difference until Command Shift  +/- is implemented in the Web Inspector.